16x9/DialogFullScreenInfo.xml

```diff
@@ -5,6 +5,7 @@
 	<onunload>Skin.Reset(PausedOverlayCast)</onunload>
 	<onunload>Skin.Reset(ShowPausedOverlayFlags)</onunload>
 	<onunload>Skin.Reset(ShowSeekbar)</onunload>
+	<onunload condition="VideoPlayer.Content(livetv) + !Skin.HasSetting(ToggleLiveTVNext)">Skin.ToggleSetting(ToggleLiveTVNext)</onunload>
```

Comment:
Why not simplify this and just reset the setting? Then you can drop the extra condition.